docs/advanced-usage/satellite-domains.mdx

```diff
@@ -54,83 +54,176 @@ There are two ways that you can configure your Clerk satellite application to wo
 - Using environment variables
 - Using properties
 
-#### Configure your satellite app with environment variables
+<Tabs items={["Environment variables", "Properties"]}>
 
+<Tab>
 You can configure your satellite application by setting the following environment variables:
 
 > In development, your publishable and secret keys will start with `pk_test_` and `sk_test` respectively.
 
-<CodeBlockTabs type="framework" options={["Next.js", "Remix"]}>
-```env filename=".env.local"
-NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
-CLERK_SECRET_KEY={{secret}}
-
-NEXT_PUBLIC_CLERK_DOMAIN=satellite.dev
-# Or, in development:
-# NEXT_PUBLIC_CLERK_DOMAIN=http://localhost:3001
-NEXT_PUBLIC_CLERK_IS_SATELLITE=true
-
-NEXT_PUBLIC_CLERK_SIGN_IN_URL=https://primary.dev/sign-in
-# Or, in development:
-# NEXT_PUBLIC_CLERK_SIGN_IN_URL=http://localhost:3000/sign-in
-```
-```env filename=".env"
-CLERK_PUBLISHABLE_KEY={{pub_key}}
-CLERK_SECRET_KEY={{secret}}
-
-CLERK_SIGN_IN_URL=http://primary.dev/sign-in
-# Or, in development:
-# CLERK_SIGN_IN_URL=http://localhost:3000/sign-in
-CLERK_DOMAIN=satellite.dev
-# Or, in development:
-# CLERK_DOMAIN=http://localhost:3001
-CLERK_IS_SATELLITE=true
-```
-</CodeBlockTabs>
-
-#### Configure your satellite app with properties
-
-You can configure your satellite application as a satellite application by setting the `isSatellite`, `domain`, and `signInUrl` properties.
+- In the `.env` file associated with your primary domain:
+    <CodeBlockTabs type="framework" options={["Next.js", "Remix"]}>
+    ```env filename=".env.local"
+    NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+    CLERK_SECRET_KEY={{secret}}
+    NEXT_PUBLIC_CLERK_SIGN_IN_URL=/sign-in
+    ```
+    ```env filename=".env"
+    CLERK_PUBLISHABLE_KEY={{pub_key}}
+    CLERK_SECRET_KEY={{secret}}
+    CLERK_SIGN_IN_URL=/sign-in
+    ```
+    </CodeBlockTabs>
+- In the `.env` file associated with your other (satellite) domain:
+    <CodeBlockTabs type="framework" options={["Next.js", "Remix"]}>
+    ```env filename=".env.local"
+    NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+    CLERK_SECRET_KEY={{secret}}
+    NEXT_PUBLIC_CLERK_IS_SATELLITE=true
+    NEXT_PUBLIC_CLERK_DOMAIN=satellite.dev
+    # Or, in development:
+    # NEXT_PUBLIC_CLERK_DOMAIN=http://localhost:3001
+    NEXT_PUBLIC_CLERK_SIGN_IN_URL=https://primary.dev/sign-in
+    # Or, in development:
+    # NEXT_PUBLIC_CLERK_SIGN_IN_URL=http://localhost:3000/sign-in
```

Comment:
The way it's formatted is super confusing 😢 what do you think about this:

(And if you like it, let's change the Remix example too)

```suggestion
   
    # Production example:
    NEXT_PUBLIC_CLERK_DOMAIN=satellite.dev
    NEXT_PUBLIC_CLERK_SIGN_IN_URL=https://primary.dev/sign-in
 
    # Development example:
    # NEXT_PUBLIC_CLERK_DOMAIN=http://localhost:3001
    # NEXT_PUBLIC_CLERK_SIGN_IN_URL=http://localhost:3000/sign-in
```